nmosauth/auth_server/token_generator.py

```diff
@@ -39,8 +39,14 @@ def get_audience(client):
         return wildcard_domain
 
     @staticmethod
-    def populate_nmos_claim(user, scope_list):
+    def populate_nmos_claim(user, scope_list, grant_type):
         nmos_claim = {}
+        if grant_type == "client_credentials":
```

Comment:
Assuming this exception is related to https://github.com/AMWA-TV/nmos-authorization-practice/pull/8/files, should it handle the events scope too?